payment/src/main/java/org/killbill/billing/payment/core/sm/PaymentAutomatonDAOHelper.java

```diff
@@ -165,7 +165,7 @@ public String getPaymentProviderPluginName(final boolean includeDeteled) throws
     }
 
     public PaymentPluginApi getPaymentPluginApi() throws PaymentApiException {
-        final String pluginName = getPaymentProviderPluginName(true);
+        final String pluginName = getPaymentProviderPluginName(false);
         return getPaymentPluginApi(pluginName);
```

Comment:
I suppose changing that value does not have any impact on our tests (they work either way?)